gh-fork-ribbon.css

```diff
@@ -21,6 +21,11 @@
   position: fixed;
 }
 
+/* Fix for Safari when printing */
+.github-fork-ribbon.printer {
```

Comment:
Thanks for the contribution! I'm not sure this is the right fix though, for a few reasons:

1. It doesn't work for me (Safari 14.0 on macOS 10.15.7). Without your fix, the banners have a _white_ background both in the print preview window and when "printed" to PDF. With your fix, the banners show the correct colour in the print preview window, but are black when "printed" to PDF.
2. Why is this a separate class? Why not just make `-webkit-print-color-adjust: exact` the default?
3. If it does need to be a separate class, I'd suggest naming the class more meaningfully. E.g. the purpose of this class is to fix colours when printing from webkit browsers, so something like `webkit-print-colors` would be more self-explanatory.

Reply:
1. got it
2. I suggest we should make it as a class so the user can have options to apply the fix or not. Not suggesting to make it globally default.
3. I prefer shorter one which is "printer" I suppose to name it "support-printer" but does not make sense. What do you suggest?